### PR TITLE
added static banner

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a Comparison Tool user, I need to understand the impact that COVID has on my education benefits so that I am aware of any changes and any actions that I need to take.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9257
## Testing done
local testing

## Screenshots

Screen Shot 2020-05-26 at 2 33 10 PM

## Acceptance criteria
- [x] The following content is displayed on the Comparison Tool Landing Page.
- [x] The content is displayed as a banner and matches DEPO stylings.
- [x] The banner is displayed between the blue header and the Comparison Tool breadcrumbs.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/containers/GiBillApp.jsx [hunk 1]
```diff
@@ -89,8 +91,11 @@ export class GiBillApp extends React.Component {
       content = this.props.children;
     }
 
+    const banner = environment.isProduction() ? '' : <Covid19Banner />;
+
     return (
       <div className="gi-app">
+        {banner}
```
Comment (dmccurdybah) [inline src/applications/gi/containers/GiBillApp.jsx:98]: Can you drop the `banner` and do `!environment.isProduction() && <Covid19Banner />`?

Comment (ksantiagomorgan6) [reply src/applications/gi/containers/GiBillApp.jsx:98]: sure

File: src/applications/gi/containers/GiBillApp.jsx [hunk 2]
```diff
@@ -99,6 +104,7 @@ export class GiBillApp extends React.Component {
                 onViewLiveVersion={this.exitPreviewMode}
               />
             )}
+
```
Comment (dmccurdybah) [inline src/applications/gi/containers/GiBillApp.jsx:107]: And no need to make whitespace changes unless this was done by the linter

Comment (ksantiagomorgan6) [reply src/applications/gi/containers/GiBillApp.jsx:107]: Not sure how that got there but I removed it

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**